R/pkg_ref_cache_source_control_url.R

```diff
@@ -0,0 +1,14 @@
+pkg_res_cache.source_control_url <- function(url){
+  url <- tolower(x$pkg_website)
+  s_url <- c("github.com", "bitbucket.com", "gitlab.com")
```

Comment:
I'm not terribly familiar with BitBucket, and it took me a bit of searching to try to find public repos through that service, but I believe they are at a .org url. Also converted to a single regex so that we don't need to loop over host names. 

```suggestion
  s_url <- "(github\\.com|bitbucket\\.org|gitlab\\.com)"
```